tools/depends/target/mdnsresponder/Makefile

```diff
@@ -1,13 +1,14 @@
 include ../../Makefile.include
-DEPS= ../../Makefile.include Makefile
+DEPS= ../../Makefile.include Makefile makefile.internal 01-android-embedded.patch
```

Comment:
other depends like python or libssh also have there patches added to ```DEPS```.
This also enables rebuild of the package if only the patch file gets changed.